config/manifests/bases/external-secrets-operator.clusterserviceversion.yaml

```diff
@@ -18,6 +18,7 @@ metadata:
     features.operators.openshift.io/token-auth-aws: "true"
     features.operators.openshift.io/token-auth-azure: "true"
     features.operators.openshift.io/token-auth-gcp: "true"
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.19"}]'
```

Comment:
you can add `olm.minOpenShiftVersion` too.

Ti's what I learned, reading through https://github.com/openshift/operator-framework-olm/blob/1c88f599825d8d3a6aa2d8aa357124101b00e31d/staging/operator-lifecycle-manager/pkg/controller/operators/openshift/helpers.go#L234.